util/ldb_cmd.h

```diff
@@ -237,20 +291,43 @@ class LDBCommand {
       }
       db_ = db_ttl_;
     } else if (is_read_only_) {
-      st = DB::OpenForReadOnly(opt, db_path_, &db_);
+      st = DB::OpenForReadOnly(opt, db_path_, column_descriptors_,
+              &column_handles_, &db_);
     } else {
-      st = DB::Open(opt, db_path_, &db_);
+      st = DB::Open(opt, db_path_, column_descriptors_, &column_handles_, &db_);
     }
     if (!st.ok()) {
       string msg = st.ToString();
       exec_state_ = LDBCommandExecuteResult::Failed(msg);
+    } else {
+      assert(column_descriptors_.size() == column_handles_.size());
+      // Validate column families passed as an argument, if any
+      ValidateColumnFamiliesArg();
     }
 
     options_ = opt;
   }
 
+  Status InitColumnFamilies() {
+    vector<string> names;
+    Status s = DB::ListColumnFamilies(DBOptions(), db_path_, &names);
+    if (!s.ok()) {
+      return s;
+    }
+    for (auto const& name : names) {
+      // Are default options ok?
```

Comment:
As a start, can we use the same options as for the default column family? Options for default column family can get modified through the command line argument.s